GitHubData/Services/IGitHubDataService.cs

```diff
@@ -2,13 +2,19 @@
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
-namespace GitHubData;
+namespace GitHubData.Services;
 
 /// <summary>
 /// Provides access to GitHub data.
 /// </summary>
 public interface IGitHubDataService : IDisposable
 {
+    /// <summary>
+    /// Gets or sets the auth token to use when making requests for GitHub data.
```

Comment:
Refactor code docs to below:

```suggestion
    /// Gets or sets the authorization token to use when making requests for GitHub data.
```